README.md

```diff
@@ -1,38 +1,82 @@
-# Newdle
+## WARNING: newdle is not yet production ready
+### Currently missing
+ * [ ] E-mail sending;
+ * [ ] Integration with calendaring systems;
+ * [ ] Integration with Indico;
 
-This is something new and cool! 🎉
+If you would like to help with any of that, please contact us through the relevant GitHub issue.
 
-## Database
+# Newdle [![Travis Build Status](https://travis-ci.com/indico/newdle.svg?branch=master)](https://travis-ci.org/indico/newdle) [![License](https://img.shields.io/github/license/indico/newdle.svg)](https://github.com/indico/newdle/blob/master/LICENSE) [![Made at CERN!](https://img.shields.io/badge/CERN-Open%20Source-%232980b9.svg)](https://home.cern)
+
+<p align="center">
```

Comment:
Is there no way to avoid putting HTML in there?!

Reply:
I don't think there is another way to do that